mne/viz/evoked.py

```diff
@@ -909,7 +916,36 @@ def plot_evoked_white(evoked, noise_cov, show=True):
            signals, vol. 108, 328-342, NeuroImage.
     """
     return _plot_evoked_white(evoked=evoked, noise_cov=noise_cov,
-                              scalings=None, rank=None, show=show)
+                              scalings=None, rank=rank, show=show)
+
+
+def _match_proj_type(proj, ch_names):
+    """See if proj should be counted."""
+    proj_ch_names = proj['data']['col_names']
+    select = any(kk in ch_names for kk in proj_ch_names)
+    return select
+
+
+def _check_estimated_rank(this_estimated_rank, this_picks, this_info, evoked,
+                          cov, ch_type, has_meg, has_sss):
+    """Compare estimated against expected rank."""
+    expected_rank = len(this_picks)
+    expected_rank_reduction = 0
+    if has_meg and has_sss and ch_type == 'meg':
+        sss_rank = _get_rank_sss(evoked)
+        expected_rank_reduction += (expected_rank - sss_rank)
+    n_ssp = sum(pp['active'] for pp in cov['projs'] if
```

Comment:
@agramfort agreed with my suggestion that we change this to just do `sum(_match_proj_type(...) for pp in cov['projs'])` since projectors are applied during inversion

Reply:
ok. Thank for the suggestion. This is very helpful.